### ISSUE TITLE
File with the same name, chrome debugger breakpoint location is confused

### ISSUE THREAD

**Issue by liningftp** [babel/babel#10786]
## Bug Report

**Current Behavior**
Open Chrome and Turn on F12
Visit http://127.0.0.1/index.html
Breakpoint occurs in ./2/a.js, line2 
```
var b = 2;
```

But should occurs in ./a.js, line2
```
debugger
```

**Input Code**

./index.html
```
<!doctype html>
<html lang="en">
    <head>  
      <meta charset="utf-8">
      <title>test</title>
    </head>
    <body>
      <script src="https://unpkg.com/@babel/standalone@7.7.4/babel.min.js"></script>
      <script type="text/babel" src="./a.js"></script>
      <script type="text/babel" src="./2/a.js"></script>
      <script type="text/babel">
      	T();
      </script>
    </body>
</html>
```

./a.js
```
function T(){
	debugger;
}
```

./2/a.js
```
var a = 1;
var b = 2;
var c = 3;
var d = 4;
```

**Expected behavior/code**
Breakpoint occurs in ./a.js

**Environment**

- Babel version(s): v7.7.4
- Node/npm version: Node 13.0.1 /npm 6.12.0
- OS: Windows 10
- Browser: Chrome 78.0.3904.108 (64 bit)
- Monorepo: [e.g. yes/no/Lerna]
- How you are using Babel: loader

**Comment by liningftp** [comment]
if ./2/a.js in the after ./a.js e.g.
```
      <script type="text/babel" src="./2/a.js"></script>
      <script type="text/babel" src="./a.js"></script>
```
debugger is right

**Comment by JLHwung** [comment]
@liningftp Good catch! What happens here is that babel-standalone generates incorrect source maps which confuses Chrome DevTools. It will be fixed in #10797.
